### ISSUE TITLE
This is incorrect: "365 Apps... only supports... (RDS) when purchased through a Volume Licensing Program"

### ISSUE THREAD

**Issue by ContreBombarde** [MicrosoftDocs/OfficeDocs-O365ServiceDescriptions#775]
_7 Microsoft 365 Apps for Enterprise is supported on user-dedicated Virtual Desktop Infrastructure (VDI). Microsoft 365 Apps for Enterprise only supports Remote Desktop Services (RDS) when purchased through a Volume Licensing Program._

This is terribly confusing, as most purchase O365 via a CSP and it may not even be possible for them to purchase it via Volume Licensing, yet it's affecting their future decisions on this. Asking a VAR licensing expert about this got a "huh?" response. Thankfully, this was clarified [here](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/issues/997) (better late than never), but this info is incorrect info is still out there (such as this page). But since it's only in the feedback page and removed from the actual documentation (rather than positively identified as _not_ necessary), one wouldn't necessarily know without digging through feedback comments that there is no requirement to purchase through a Volume Licensing Program. 

Hopefully this page will be updated as a result, but in the meantime, hopefully this note will help others with the same concern. 

Thank you for allowing the community to offer public feedback--it is immensely helpful!

**Comment by SeanGit88** [comment]
Thanks for your comment. I've forwarded your query to the product team and will update the article as necessary based on their response.

**Comment by ContreBombarde** [comment]
Thank you, Sean!

**Comment by SeanGit88** [comment]
Hello, this has been updated. thanks for your feedback.

**Comment by ContreBombarde** [comment]
Thank you, @SeanGit88, but I think perhaps the wrong part was updated, unless the change hasn't been committed yet. This part remains:

"Microsoft 365 Apps for Enterprise only supports Remote Desktop Services (RDS) **when purchased through a Volume Licensing Program**."

But [this ](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/issues/997)clarifies that purchase through a Volume Licensing Program is not a requirement:

"Purchase through a volume license program isn't required. We've removed that from the article. Thanks for pointing that out to us."
